src/tests/system/playground-kyber.systest.ts

```diff
@@ -36,7 +41,7 @@ describe('playground', () => {
 
     const manager = withNewAccount(master);
     const trader = withNewAccount(master);
-
+    console.log('Manager address is: ', manager.wallet.address);
```

Comment:
I suggest to use `log.debug` to have it in the log files too and is not swallowed by jest.